logstreams/src/main/java/io/zeebe/logstreams/log/LogStreamReader.java

```diff
@@ -66,20 +66,12 @@
   /**
    * The last log storage address, from which the last block of events was read.
    *
-   * <p/>
-   * Useful if you want to found out the related block address, then just seek to a given position
-   * and call {@link #lastReadAddress).
+   * <p>Useful if you want to found out the related block address, then just seek to a given
```

Comment:
Why we should ever want this :laughing:

Reply:
Good question :sweat_smile: I think this is called for deletion? :man_shrugging:

Reply:
Should probably not be part of the public api :laughing: but we refactor this later